miqbox/miqbox.py

```diff
@@ -494,8 +494,9 @@ def kill(connection, name):
 @cli.command()
 @click.option("-l", "--local", is_flag=True, help="All available local images")
 @click.option("-r", "--remote", is_flag=True, help="All available remote images")
+@click.option("-f", "--find", type=str, help="Find specific image")
```

Comment:
@Yadnyawalkya Just one think... I would use `filter` as command than `find`. What you say?

Reply:
`filter` suite here. :100: